packages/date-time/src/components/DatePicker/DatePicker.tsx

```diff
@@ -0,0 +1,16 @@
+import { styled } from '@uifabric/utilities';
+import { IDatePickerProps, IDatePickerStyleProps, IDatePickerStyles } from './DatePicker.types';
+import { DatePickerBase } from './DatePicker.base';
+import { styles } from './DatePicker.styles';
+
+/**
+ * DatePicker description
+ */
+export const DatePicker: React.StatelessComponent<IDatePickerProps> = styled<IDatePickerProps, IDatePickerStyleProps, IDatePickerStyles>(
```

Comment:
Isn't the compiler able to figure out these types without having them explicitly declared?

Reply:
I'll take note of these and mark them as items todo a little later- this is a direct copy-paste of the existing DatePicker control in office-ui-fabric-react. :)